### PR TITLE
Update cryptography and .travis.yml [CLOUDDST-7308]

### PR DESCRIPTION
**Pull request by pbortlov**
commands used for generating the upgrade:

- podman exec iib_iib-worker_1 pip install --upgrade pip pip-tools
- podman exec iib_iib-worker_1 pip-compile --generate-hashes --output-file=requirements.txt --upgrade-package "cryptography"
- podman exec iib_iib-worker_1 pip-compile --generate-hashes --output-file=requirements-test.txt requirements-test.in

### CODE REVIEW

**Comment by pbortlov** [comment]
This PR can be merged after PRs in iib-worker and iib-web ansible-roles

**Review by midnightercz — APPROVED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -100,14 +108,6 @@ flask-sqlalchemy==2.4.1 \
     # via
     #   flask-migrate
     #   iib (setup.py)
-flask==1.1.1 \
```
Comment (yashvardhannanavati) [inline requirements.txt:103]: Why was this changed? Automatically or was this done manually?  It's the exact same as before just in a different position.

File: requirements.txt [hunk 2]
```diff
@@ -260,20 +260,26 @@ pytz==2019.3 \
     --hash=sha256:1c557d7d0e871de1f5ccd5833f60fb2550652da6be2693c1e02300743d21500d \
     --hash=sha256:b02c06db6cf09c12dd25137e563b31700d3b80fcc4ad23abb7a315f2789819be
     # via celery
-requests-kerberos==0.12.0 \
```
Comment (yashvardhannanavati) [inline requirements.txt:263]: Same as above

File: requirements.txt [hunk 3]
```diff
@@ -295,17 +301,10 @@ ruamel.yaml.clib==0.2.0 \
     --hash=sha256:ed5b3698a2bb241b7f5cbbe277eaa7fe48b07a58784fba4f75224fd066d253ad \
     --hash=sha256:f9dcc1ae73f36e8059589b601e8e4776b9976effd76c21ad6a855a74318efd6e
     # via ruamel.yaml
-ruamel.yaml==0.16.10 \
```
Comment (yashvardhannanavati) [inline requirements.txt:298]: THis too

File: requirements.txt (hunk 1 above)
Comment (pbortlov) [reply requirements.txt:103]: It is changed because somebody generate this file with a different python version before. You can check line no.2 and it is changed `with python 3.6`

Comment (pbortlov) [reply requirements.txt:103]: this is automatically generated.

**Review by midnightercz — DISMISSED** [review]

**Review by lipoja — APPROVED** [review]

**Review by midnightercz — APPROVED** [review]

**Review by yashvardhannanavati — APPROVED** [review]
